addons/im_livechat/static/src/core/web/messaging_menu_patch.js

```diff
@@ -1,24 +1,28 @@
 /* @odoo-module */
 
+import { Store } from "@mail/core/common/store_service";
 import { MessagingMenu } from "@mail/core/web/messaging_menu";
 
 import { _t } from "@web/core/l10n/translation";
 import { patch } from "@web/core/utils/patch";
 
-patch(MessagingMenu.prototype, {
+patch(Store.prototype, {
```

Comment:
This should be moved to a store patch file.

Also note: does messaging menu have facade to new code?